tables/requirements_blitz.txt

```diff
@@ -0,0 +1,95 @@
+.def party2b character1&character3
+.def party2c character1&character4
+.def party2d character2&character3
+.def party2e character2&character4
+.def party2f character3&character4
+.def party2 party2a|party2b|party2c|party2d|party2e|party2f
+
+.def party3a character1&character2&character3
+.def party3b character1&character2&character4
+.def party3c character1&character3&character4
+.def party3d character2&character3&character4
+.def party3 party3a|party3b|party3c|party3d
+
+.def party4 character1&character2&character3&character4&all_capsules
+
+.def airship engine
+.def submarine mermaid_jade
+.def submarine_or_airship submarine|airship
+.def victory victory
+
+.def dankirk hook&bomb&dankirk_key&party3
+.def gratze dankirk|submarine_or_airship
+
+starting_character  *
+starting_item       *
+hidden_item         *
+skill_cave          arrowlike&party0
+foomy_woods         party0
+darbi_shrine        party0
+zeppy_cave          submarine&party0
+sundletan_cave      party0
+catfish_cave        arrowlike&lake_key&party0
+alunze_cave         breakable&party0
+alunze_basement     breakable&party0
+tanbel_tower1       sky_key&party1
+tanbel_tower2       sky_key&party1
+ruby_cave           ruby_key&party1
+ruby_capsule        breakable&party0
+tsword_shrine1      bomb&sword_key&party1
+tsword_shrine2      bomb&sword_key&party1
+gordovan_tower1     hook&bomb&wind_key&party1
+gordovan_tower2     hook&bomb&wind_key&party1
+cave_bridge         breakable&hook&party0
+north_dungeon       hook&bomb&party1
+north_capsule       hook&party1
+ancient_tower1      hook&bomb&cloud_key&party1
+ancient_tower2      hook&bomb&cloud_key&party1
+lighthouse1         light_key&party1
+lighthouse2         light_key&party1
+phantom_mountain    hook&bomb&fire_arrow&tree_key&party2
+sacrifice_tower1    hook&bomb&narcysus_key&party2
+sacrifice_tower2    hook&bomb&narcysus_key&party2
+sacrifice_capsule   hook&breakable&narcysus_key&party2
+karlloon_shrine1    hook&bomb&party2
+karlloon_shrine2    hook&bomb&party2
+lexis_lab           party0
+flower_mountain     hammer&flower_key&party2
+flower_capsule      hammer&hook&party2
+dankirk_dungeon     dankirk
+northeast_tower1    hook&hammer&trial_key&party2
+northeast_tower2    hook&hammer&trial_key&party2
+no_return_mountain  party0
+divine_shrine1      hook&hammer&heart_key&party2
+divine_shrine2      hook&hammer&heart_key&party2
+vengeance_shrine1   submarine&bomb&hammer&ghost_key&party2
+vengeance_shrine2   submarine&bomb&hammer&ghost_key&party2
+truth_tower1        breakable&hook&truth_key&party2
+truth_tower2        breakable&hook&truth_key&party2
+dragon_mountain     submarine_or_airship&hook&hammer&fire_arrow&magma_key&party2
+underwater_shrine   submarine&party0
+gratze_basement     gratze&arrowlike&basement_key&party2
+shuman              airship&party3
+stradha             airship&party3
+kamirno             airship&party3
+lisa				arrowlike
+marie				breakable
+clare				party1
+daos_shrine         lisa&marie&clare&any_key
```

Comment:
It is not possible to fly to Doom Island without the engine; you should either add the engine back as a requirement to reach Daos Shrine, or create a modified event that does not require the engine.